etc/ci/performance/README.md

```diff
@@ -7,27 +7,21 @@ Servo Page Load Time Test
 
 # Basic Usage
 
-## Prepare the test runner
+If you want to run a performance test for your servo build. Simply go to the top-level servo directory and run `./mach test-perf`. The test result JOSN will be saved to `etc/ci/performance/output/`. You can compare two test results with the `test_differ.py` script. Run `python test_differ.py -h` for instructions.
```

Comment:
Nit: Fixing style,

"`./mach test-perf` can be used to run a performance test on your servo build. The test result JSON will be saved to `etc/ci/performance/output/`. You can then run `python test_differ.py` to compare these two test results. Run `python test_differ.py -h` for instructions."